src/utils.js

```diff
@@ -54,8 +55,27 @@ const fetchEndpoints = async () => {
   return data;
 };
 
+const checkForSdkUpdates = async () => {
+  try {
+    const { response } = await RESTController.ajax(
+      'GET',
+      'https://registry.npmjs.org/-/v1/search?text=moralis&size=1'
+    );
+    const latestVersion = response.objects[0].package.version;
+    const installedVersion = process.env.npm_package_version;
+    if (installedVersion < latestVersion)
+      // eslint-disable-next-line no-console
+      console.warn(
+        'You are not using the latest version of the SDK. Please update it as soon as possible to enjoy the newest features.'
+      );
+  } catch (error) {
+    throw new Error('Could not verify SDK version');
```

Comment:
Let's not throw an error but show a console warn here.
If npm is down, then we don't want all moralis apps to crash on startup
```suggestion
    // eslint-disable-next-line no-console
    console.warn('Could not verify SDK version');
```